### ISSUE TITLE
compared to which-polygon

### ISSUE THREAD

**Issue by andrewharvey** [pelias/polygon-lookup#48]
A section of the readme which explains how this compared to https://github.com/mapbox/which-polygon would be helpful.

https://github.com/mapbox/which-polygon/issues/15

**Comment by missinglink** [comment]
Hi @andrewharvey, I've not used that Mapbox library but would also be interested in the similarities/differences, would you be up for writing that comparison?

**Comment by andrewharvey** [comment]
Yeah I might work on a comparison, in the mean time I did a benchmark using 15,000 polygons, and 1000 point queries.

| library | time |
| --- | --- |
| pelias/polygon-lookup | 19ms |
| mapbox/which-polygon | 19ms |
| crosscutio/fast-point-in-poly | 74,000ms |
